### PR TITLE
[usage-collection] add pandas to requirements

### PR DESCRIPTION
**Pull request by TheRealHaoLiu**
##### SUMMARY
requested by for usage-collection work

##### ISSUE TYPE

 - Breaking Change 
 - New or Enhanced Feature
 - Bug, Docs Fix or other nominal change

##### COMPONENT NAME

 - Other

##### AWX VERSION

```
awx: 21.12.1.dev12+g0e3d9ff97d
```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

File: requirements/requirements.txt [hunk 1]
```diff
@@ -241,6 +241,8 @@ multidict==6.0.2
     #   yarl
 netaddr==0.8.0
     # via pyrad
+numpy==1.24.2
+    # via pandas
```
Comment (AlanCoding) [inline requirements/requirements.txt:245]: check is failing on

> license for requirement numpy is missing

**Comment by AlanCoding** [comment]
How is this going to be used?

**Comment by AlanCoding** [comment]
And it seems like it would be best to create the PR against the feature branch https://github.com/ansible/awx/tree/feature_usage-collection

**Comment by AlanCoding** [comment]
Link https://github.com/ansible/awx/pull/13585

**Comment by simaishi** [comment]
This fails for offline build. numpy pins setuptools to 59.2.0 and wheel to 0.37.0, which conflicts with version of setuptools/wheel used in requirements.txt.

**Comment by slemrmartin** [comment]
pandas has been replaced by csv, I suppose this PR can be closed

**Pull request closed without merging**